### ISSUE TITLE
[Provide seminars project] Training content

### ISSUE THREAD

**Issue by pdgonzalez872** [erlef/education-wg#19]
Let's start gathering content for our trainings.

**Comment by pdgonzalez872** [comment]
@costaraphael, you mentioned you had some content right? Could you please share some of the content you have?

@elbrujohalcon can you share what you will present at the meetup?

@ferd Do you have anything that can be shared as well?

**Comment by ferd** [comment]
well considering we just had a meeting where it was mentioned we may want to focus on elixir stuff, I have nothing.

**Comment by pdgonzalez872** [comment]
@ferd please don't let that stop you. At this point, I don't think we can restrict anything, we welcome all content. I personally think Erlang material would be fantastic.

**Comment by elbrujohalcon** [comment]
@pdgonzalez872 I don't yet have the slides ready for sharing.
I have the slide deck, but it's private to NextRoll… I have to prepare new ones (likely based on the ones I used for NextRoll), but they're not ready just yet.

**Comment by elbrujohalcon** [comment]
My plan was to make them public **after** the meetup.

**Comment by pdgonzalez872** [comment]
@elbrujohalcon no worries, whenever you are ready would be good! :) Thanks for doing this.

**Comment by elbrujohalcon** [comment]
So, I have to _Elixir-ize_ it… and I have to fix the exercises because they're built _on top_ of some previous training, but… for a rough idea, [this](https://docs.google.com/presentation/d/1VQTS6TMapaPZpWI_HF6dpnDro_nPmUOSfBRLdFzXEjE/edit#slide=id.g4f2cd92d7e_0_27) is what I'm preparing for the BCN Meetup Intervention

**Comment by ferd** [comment]
re: exceptions, I like to stick to the words "raise exceptions" to make a distinction between `throw` and the other exception things; then it acts as a decent reminder for `erlang:raise/3` being usable for all kinds of exceptions.

**Comment by elbrujohalcon** [comment]
Good point, @ferd … I'll enable commenting on the slides for our group members so you can all share your insights directly on the slides.
